public/css/style.css

```diff
@@ -0,0 +1,52 @@
+body {
```

Comment:
nice look!

Reply:
Haha, thank you. Not a professional dev yet but I will at least try to make it presentable for you to look at ^^